app/controllers/concerns/concerns/load_settings.rb

```diff
@@ -0,0 +1,32 @@
+module Concerns
```

Comment:
why `concerns/concerns` ? I think that concerns aren't usually put under a `Concerns` module